client/src/components/posting-page/JobPosting.js

```diff
@@ -182,10 +184,20 @@ const ErrorMessage = ({ errorMessage }) => {
   )
 }
 
-const HiddenNotification = ({ jobPosting }) => {
+const HiddenNotification = ({ showFrom, showTo }) => {
+  let visibilityMessage = 'Permanently hidden'
+
+  if (showFrom) {
```

Comment:
I think this could be written `if (showTo) { .... } else { .... }`. It's really just a difference in preference, but I think if-else is easier to read than nested ifs. 🙂